src/wrapper/quabasevariable.cpp

```diff
@@ -270,6 +270,7 @@ void QUaBaseVariable::setValue(
 	// get modifiable copies
 	auto newValue = value;
 	auto newType  = newTypeConst;
+	auto originalType  = (QMetaType::Type)value.type();

```

Comment:
You think we can move this inside the `#else`, since is seems to be Qt6-specific? (And same with the others)

Reply:
Yes, it is Qt6-specific. Let me update it.